framework/components/ecl-files/ecl-files.twig

```diff
@@ -1,112 +1,144 @@
-<!-- Default -->
-<div class="file">
-  <span class="icon icon--file file__icon"></span>
-  <div class="file__metadata">
-    <span class="file__title">File title example</span>
-    <div class="file__info">
-      <span class="file__content-language">English</span> (213.25 kB - PDF - 4 pages)
-    </div>
-  </div>
-  <a href="#" class="btn btn-default file__btn">
-    Download<span class="ecl-sr-only">(213.25 kB - PDF - 4 pages)</span>
-  </a>
-</div>
-
-<!-- Bare -->
-<div class="file file--bare">
-  <span class="icon icon--file file__icon"></span>
-  <div class="file__metadata">
-    <span class="file__title">File title example</span>
-    <div class="file__info">
-      <span class="file__content-language">English</span> (213.25 kB - PDF - 4 pages)
+{% if variant is defined %}
+
+  {% if variant == "translations" %}
+
+<!-- Translations -->
+<section class="ecl-file">
+  <div class="ecl-file__body">
+    <div class="ecl-row">
+      <div class=" ecl-col ecl-col-md-8">
+        <span class="ecl-icon ecl-icon--{{ icon }} ecl-file__icon"></span>
+        <div class="ecl-file__metadata">
+          <div class="ecl-file__title">{{ title }}</div>
+          <div class="ecl-file__info">
+            <span class="ecl-file__language">{{ language }}</span> ({{ meta }})
+          </div>
+        </div>
+      </div>
+
+      <div class="ecl-col-12 ecl-col-md-4">
+        {% include '@ec-europa/ecl-buttons' with {
+          'label': buttonLabel ~ '<span class="ecl-sr-only">(' ~ meta ~ ')',
+          'isBlock': true,
+          'extraClass': 'ecl-file__download',
+          'extraAttributes': [
+            { 'name': 'type', 'value': 'button' },
+          ]
+        } %}
+      </div>
     </div>
   </div>
-  <a href="#" class="btn file__btn btn--condensed is-internal">
-    Download<span class="ecl-sr-only">(213.25 kB - PDF - 4 pages)</span>
-  </a>
-</div>
-
-<!-- Image -->
-<div style="width: 450px;"> <!-- only required for the style guide -->
-  <div class="file file--image">
```

Comment:
yes, indeed. We will have to remember this when upgrading the banner component.
I added a comment in dedicated ticket